packages/devtools_app/pubspec.yaml

```diff
@@ -17,9 +17,8 @@ resolution: workspace
 dependencies:
   ansi_up: ^1.0.0
   ansicolor: ^2.0.0
-  async: ^2.0.0
-  clock: ^1.1.1
-  codicon: ^1.0.0
+  async: ^2.0.0 # okay
```

Comment:
The 'okay' text was some local bookkeeping?

Reply:
Yeah this was an accidental commit. Feel free to remove as part of https://github.com/flutter/devtools/pull/8599 or I can submit a clean up PR.